packages/components/src/font-size-picker/stories/index.tsx

```diff
@@ -8,37 +13,22 @@ import { useState } from '@wordpress/element';
  */
 import FontSizePicker from '../';
 
-export default {
+const meta: ComponentMeta< typeof FontSizePicker > = {
 	title: 'Components/FontSizePicker',
 	component: FontSizePicker,
-	argTypes: {
-		initialValue: { table: { disable: true } }, // hide prop because it's not actually part of FontSizePicker
-		fallbackFontSize: {
-			description:
-				'If no value exists, this prop defines the starting position for the font size picker slider. Only relevant if `withSlider` is `true`.',
-		},
-		size: {
-			control: { type: 'radio' },
-			options: [ 'default', '__unstable-large' ],
-		},
-		withReset: {
-			description:
-				'If `true`, a reset button will be displayed alongside the input field when a custom font size is active. Has no effect when `disableCustomFontSizes` or `withSlider` is `true`.',
-			control: { type: 'boolean' },
-			table: {
-				type: 'boolean',
-				defaultValue: { summary: true },
-			},
-		},
-	},
+	argTypes: {},
 	parameters: {
 		controls: { expanded: true },
 		docs: { source: { state: 'open' } },
 	},
```

Comment:
Let's leverage Storybook's actions

We can do so by flagging `on*` callbacks as actions, and passing through the `onChange` prop in `FontSizePickerWithState`:

```diff
diff --git a/packages/components/src/font-size-picker/stories/index.tsx b/packages/components/src/font-size-picker/stories/index.tsx
index 4937214fca..51bc3c3831 100644
--- a/packages/components/src/font-size-picker/stories/index.tsx
+++ b/packages/components/src/font-size-picker/stories/index.tsx
@@ -16,8 +16,11 @@ import FontSizePicker from '../';
 const meta: ComponentMeta< typeof FontSizePicker > = {
 	title: 'Components/FontSizePicker',
 	component: FontSizePicker,
-	argTypes: {},
+	argTypes: {
+		value: { control: { type: null } },
+	},
 	parameters: {
+		actions: { argTypesRegex: '^on.*' },
 		controls: { expanded: true },
 		docs: { source: { state: 'open' } },
 	},
@@ -26,6 +29,7 @@ export default meta;
 
 const FontSizePickerWithState: ComponentStory< typeof FontSizePicker > = ( {
 	value,
+	onChange,
 	...props
 } ) => {
 	const [ fontSize, setFontSize ] = useState( value );
@@ -33,7 +37,10 @@ const FontSizePickerWithState: ComponentStory< typeof FontSizePicker > = ( {
 		<FontSizePicker
 			{ ...props }
 			value={ fontSize }
-			onChange={ setFontSize }
+			onChange={ ( v ) => {
+				setFontSize( v );
+				onChange?.( v );
+			} }
 		/>
 	);
 };
```

Reply:
You are a fountain of knowledge, Marco!